Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Use ARG to define the node version
+ARG NODE_VERSION
+
+# Use the Node.js version specified by NODE_VERSION
+FROM node:${NODE_VERSION}-alpine
+
+# Set working directory
+WORKDIR /app
+
+# Copy package.json and package-lock.json
+COPY package*.json ./
+
+# Install dependencies
+RUN npm ci
+
+# Copy the rest of the application code
+COPY . .
```

Comment:
doesn't this copy `node_modules` folder over, overriding what you installed with `npm ci` before?

Reply:
This could be fixed by .dockerignore as Eray mentioned on previous comments

Reply:
New commits should fix this